build.cake

```diff
@@ -81,6 +81,37 @@ var packages = new string[]{
 
 Setup(context =>
 {
+    if (isAppveyor)
```

Comment:
Moved this to make it consistent with other projects and because setting variables belongs in a setup method, not a build task that may not be called.